src/zjs_fs.c

```diff
@@ -992,9 +992,10 @@ static jerry_value_t zjs_write_file(const jerry_value_t function_obj,
 Finished:
 #ifdef ZJS_FS_ASYNC_APIS
     if (async) {
+        ZVAL err = jerry_create_number(error);
+
```

Comment:
Kind of excessive vertical whitespace throughout here. Every line doesn't need to be its own paragraph. :) I'd remove some.